crypto/x509v3/v3_cpols.c

```diff
@@ -413,7 +413,8 @@ static void print_notice(BIO *out, USERNOTICE *notice, int indent)
             num = sk_ASN1_INTEGER_value(ref->noticenos, i);
             if (i)
                 BIO_puts(out, ", ");
-            tmp = i2s_ASN1_INTEGER(NULL, num);
+            if ((tmp = i2s_ASN1_INTEGER(NULL, num)) == NULL)
+                return;
```

Comment:
> Is it possible to trigger an i2s_ASN1_INTEGER failure through invalid input? It will only normally fail due to a malloc failure.

Yes I think it is possible the trigger this if num = sk_ASN1_INTEGER_value(ref->noticenos, i);
above returns NULL.